Content.Server/Flash/FlashSystem.cs

```diff
@@ -165,7 +165,7 @@ public void FlashArea(EntityUid source, EntityUid? user, float range, float dura
             foreach (var entity in flashableEntities)
             {
                 // Check for unobstructed entities while ignoring the mobs with flashable components.
-                if (!_interactionSystem.InRangeUnobstructed(entity, mapPosition, range, CollisionGroup.Opaque, (e) => flashableEntities.Contains(e)))
+                if (!_interactionSystem.InRangeUnobstructed(entity, mapPosition, range, CollisionGroup.Opaque, (e) => flashableEntities.Contains(e) || e == source))
```

Comment:
```suggestion
                if (e == source || !_interactionSystem.InRangeUnobstructed(entity, mapPosition, range, CollisionGroup.Opaque, (e) => flashableEntities.Contains(e)))
```
minor optimization.

Reply:
```e``` is not defined there, maybe you meant entity? But still not the same thing.

Reply:
You have to pass it to InRangeUnobstructed to make sure it's ignored by the raycast.

Reply:
ah, i'm silly. nvm then